docs/src/app/components/pages/components/date-picker.jsx

```diff
@@ -30,6 +30,12 @@ export default class DatePickerPage extends React.Component {
         name: 'Props',
         infoArray: [
           {
+            name: 'container',
+            type: 'one of: dialog, container',
+            header: 'default: dialog',
+            desc: 'The date pickers container type',
```

Comment:
This description is a bit terse - what does "inline" actually mean? (Also it's "The date picker's container type", or "Container type of the date picker" if you prefer...)